apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/peer/Peers.java

```diff
@@ -407,12 +409,19 @@ public static void suspend() {
         if(peerHttpServer!=null){
            suspend = peerHttpServer.suspend();
         }
+        TaskDispatcher dispatcher = taskDispatchManager.getDispatcher(BACKGROUND_SERVICE_NAME);
```

Comment:
Do we need 'resume()' method ?
I think we need to add calling both such Peers methods in ShardObserver class.